docs/guides/setting-up-a-networked-meta-avatar.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: docs
+title: Setting up a Networked Meta Avatar
+description: This guide walks you through setting up a networked meta avatar.
+
+---
+
+# Setting up a Networked Meta Avatar
+
+This guide will demonstrate how to quickly load a Meta Avatar and network it to other machines using Normal.
+
+When you're finished, you'll end up with an application that will let you see and talk to other player's avatars:
+
+![](./setting-up-a-networked-meta-avatar/player-wave.png "Meta Avatar waving at the viewer") 
+
+## Setup your App for Meta Avatars
+
+Before your app can load avatar data from Meta's servers, it will need permission to do so.  Ensure that your app is [configured as required for Meta Avatar usage](https://developer.oculus.com/documentation/unity/meta-avatars-app-config/), and that you have set your Oculus Platform Settings to point at the right app IDs
+
+## Setting up your VR environment
+
+Start by creating a blank unity VR scene, with the camera properly tracked to the player's head.
+
```

Comment:
I feel like if someone is beginner enough to need to be told to set up a camera that tracks the player's head, we should also explain how to do that? Showing a screenshot with the TrackedPoseDriver isn't going to be immediately obvious to a beginner that you need to add that component in order to make the camera track the head imo.

OculusSetFloorTracking is really a component that exists for the example scenes so they don't need to depend on the Oculus plugin. However, in this case, it's not possible to use Meta Avatars without the Oculus plugins. I don't think we should be telling folks to use it this component. They should configure this through OVRManager when they set that up later on in this guide.

Reply:
Swapped out the camera picture for a 2021 version which doesn't have the OculusSetFloorTracking.